common/lib/xmodule/xmodule/library_content_module.py

```diff
@@ -176,7 +192,16 @@ def make_selection(cls, selected, children, max_count, mode):
             else:
                 raise NotImplementedError("Unsupported mode.")
             selected |= added_block_keys
-
+        logger.warning(
+            "EDUCATOR-1290: LibraryContentModule.make_selection executed. "
+            "selected: {0} | invalid: {1} | overlimit: {2} | added: {3} | log ID: {4}".format(
+                selected,
```

Comment:
You can also log `valid_block_keys` here so you don't need to do the logging on line 165.